lib/puppet_x/cisco/aci_provider_utils.rb

```diff
@@ -144,53 +144,54 @@ def self.conv_puppet_attrs_2_mo_attrs(p_inst, puppet_attrs)
       #
-      #@param p_inst [ Object] Provider instance
-      #@return [JSON] payload for APIC REST request for POST
+      # @param p_inst [ Object] Provider instance
+      # @return [JSON] payload for APIC REST request for POST
       #
       def self.gen_json4createmodify(p_inst)
         attributes = p_inst.property_flush.clone
-        attributes.delete_if {|key,value| (key == :ensure ||
-                                  key == :provider ||
-                                  p_inst.class.parent_namevars.include?(key)) }
+        attributes.delete_if do |key, _value|
+          (key == :ensure || key == :provider ||
+            p_inst.class.parent_namevars.include?(key))
+        end
         mo_attributes = conv_puppet_attrs_2_mo_attrs(p_inst, attributes)
-        payload = {p_inst.class.mo_class_name => {"attributes" => mo_attributes}}
+        payload = { p_inst.class.mo_class_name => { 'attributes' => mo_attributes } }
         payload.to_json
       end
 
-      #Debug routine . Prints the difference between two hash
-      #Note: Each provider instance is associated with a single MO
+      # Debug routine . Prints the difference between two hash
+      # Note: Each provider instance is associated with a single MO
       #
-      #@param updated_attrs [Hash] Puppet attributes after operation
-      #@param orig_attrs [Hash] Puppet attributes before operation
+      # @param updated_attrs [Hash] Puppet attributes after operation
+      # @param orig_attrs [Hash] Puppet attributes before operation
       #
       def self.debug_what_changed(updated_attrs,
                                   orig_attrs)
-        updated_attrs.each do |key,value|
+        updated_attrs.each do |key, value|
           if orig_attrs[key] != value
             puts "Changed '#{key}' from= '#{orig_attrs[key]}' to= '#{value}' "
           end
         end
       end
 
-      #Set the property_flush has with necessary key values from source hash
-      #Note: Each provider instance is associated with a single MO
+      # Set the property_flush has with necessary key values from source hash
+      # Note: Each provider instance is associated with a single MO
       #
-      #@param p_inst [Object] Provider instance
+      # @param p_inst [Object] Provider instance
       #
-      def self.set_property_flush_keys_from_property_hash(p_inst)
+      def self.property_flush_keys_from_property_hash(p_inst)
         p_inst.class.allnamevars.each do |attr|
           p_inst.property_flush[attr] = p_inst.property_hash[attr]
         end
       end
 
-      #Update property_flush with attributes from a source hash
+      # Update property_flush with attributes from a source hash
       #
-      #@param p_inst [Object] Provider instance
+      # @param p_inst [Object] Provider instance
       #
-      def self.set_property_flush_attrs_frm_resource(p_inst)
+      def self.property_flush_attrs_frm_resource(p_inst)
```

Comment:
Why did the method name change here?

Reply:
Rubocop didn't like methods starting with get_ and set_ .